src/app/core/library-module/library.service.ts

```diff
@@ -113,7 +116,7 @@ export class LibraryService {
     // Show loading spinner
     this._loading$.next(true);
     // Url route for bundling
-    const bundle = USER_ROUTES.OBJECT_BUNDLE(
+    const bundle = BUNDLING_ROUTES.BUNDLE_LEARNING_OBJECT(
```

Comment:
Why is bundling in library?

Reply:
Your guess is as good as mine. Client needs to be refactored

Reply:
History: Library used to have a download button and we used to bundle everything on the fly.